### PR TITLE
feat(env): install Node.js using a version range

### PR DESCRIPTION
**Pull request by zkochan**

### CODE REVIEW

File: packages/plugin-commands-env/test/env.test.ts [hunk 1]
```diff
@@ -25,3 +26,37 @@ test('install node', async () => {
...
+
+  const { stdout } = execa.sync('node', ['-v'], {
+    env: {
+      [PATH]: `${process.cwd()}${path.delimiter}${process.env[PATH] as string}`,
+    },
+  })
+  expect(stdout.toString()).toBe('v6.17.1')
+
+  const dirs = fs.readdirSync(path.resolve('nodejs'))
+  expect(dirs).toEqual(['6.17.1'])
```
Comment (juanpicado) [inline packages/plugin-commands-env/test/env.test.ts:48]: could users download any Node.js version?

Comment (zkochan) [reply packages/plugin-commands-env/test/env.test.ts:48]: Any of those that are in this index: https://nodejs.org/download/release/index.json

Comment (juanpicado) [reply packages/plugin-commands-env/test/env.test.ts:48]: I see, the point is being able to download any Node.js version https://github.com/pnpm/pnpm/discussions/3434 but the bundle one is something already predefined in build time.

Comment (zkochan) [reply packages/plugin-commands-env/test/env.test.ts:48]: Yes, the bundled one is static. The bundled one is only used to run the pnpm CLI

**Comment by juanpicado** [comment]
@zkochan what's the command to test it?

**Comment by zkochan** [comment]
`pnpm env use -g lts`

**Review by juanpicado — APPROVED** [review]
LGTM

**Pull request merged**